src/adaptation/icap/ModXact.cc

```diff
@@ -1193,10 +1193,10 @@ void Adaptation::Icap::ModXact::parseBody()
         return;
     }
 
-    debugs(93,3,HERE << this << " needsMoreData = " << bodyParser->needsMoreData());
+    debugs(93,3,this << " needsMoreData = " << bodyParser->needsMoreData());
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.